Documentation/Signoffs/SensorSignoff.md

```diff
@@ -0,0 +1,133 @@
+
+6. For every sensor that is plugged onto the Air Quality Device initially, there may be a start up time for initial calibration. All sensors shall have a wait/warm-up time less than 2 hours before data acquisition.
+
+7. According to the ANSI/ASHRAE Standard 62.1-2016: Ventilation for Acceptable Indoor Air Quality [2], indoor CO2 concentrations no greater than 700 parts per million (ppm) above outdoor CO2 concentrations will satisfy a substantial majority (about 80%) of occupants. Sensors used for this device shall comply with their pollutant concentration with this standard for indoor/outdoor use.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; O3 - At least 0.08 parts per million inside and 0.115 ppm outside [8].
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SO2 - At least 0.02 ppm inside and 0.14 outside annually [9].
+
+## BUILDABLES OF THE SUBSYSTEM
+
+![CAPSTONE II Sensor Subsystem](https://user-images.githubusercontent.com/118767661/220430088-dc5df317-ef82-4628-a876-0c82efdccfef.png)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Figure 1. Sensor Subsystem through Interface Schematic
+
+![Screenshot (229)](https://user-images.githubusercontent.com/118767661/220430812-1e9725cd-139a-45dc-94f5-ec22bebd58c4.png)
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Figure 2. Sample code for the MIKROE-2767
+
+![Screenshot (231)](https://user-images.githubusercontent.com/118767661/220431271-221e77c1-a7d6-4de3-9dff-db658f13ef42.png)
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Figure 3. Sample Code for the A02P43B7001
+
+## ANALYSIS OF THE SUBSYSTEM
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The team has chosen 3 sensors for testing that matches with the constraints list. We wanted each to be a different type of sensor output to test modularity and implement the necessary actions required within the software to model the functionality of both the Microcontroller, Interface, and Sensor Subsystems. All sensors chosen for experimentation are for detecting one of the 6 pollutants that the proposal has stated for data input.
+
+### ANALOG VOLTAGE SENSOR
+
```

Comment:
List sensors by what they measure, not by how they output the data. If you want to also list the output type put that in parenthesis, e.g. "Temperature Sensor (analog current output)."

For the analysis of each sensor use corresponding numbers from the constraints (it's easier to read if this is done instead of bullets).

At least one sensor for each of the pollutants listed in constraint #1 needs to be identified and analyzed (but not all will be ordered at this time). Include all of them in the parts list but for the ones not needed right now set the ordering quantity to zero, along with a footnote that they will not be ordered at this time.

The ULPSM-IAQ 968-008 and A02P43B7001 is not clear. Are both of these required to detect this pollutant? Also these are relatively costly so it is desirable to find something less expensive to use for the initial order.

Some of the references are not at a suitable level, for example [11]. This content of this reference is something that would be considered common knowledge to anybody with an EE or CmpE degree.

Reply:
Sensor titles have been changed.

I left different brackets at the end of each bullet to signify the corresponding constraint with its number inside. Curly brackets are for constraints, and the squared brackets are for the corresponding reference. Since I have the most options to select, creating another numbered list may seem confusing to the reader.

Okay I will make the edits, this change will also be shown within the schematic. Could not find a lead monoxide sensor. Therefore, I'll be changing it to methane instead.

Both would be used to detect and output measurements of the pollutant. As started in the project proposal, the budget did not include the sensors, because of the various types the user may use. If you prefer, I can try to find a cheaper one.

I used the reference to support the point that the interface subsystem is needed for analog output. If you would like, I can take it off.

Reply:
Move the note about ordering zero of some sensors to the BOM section. It is not relevant for the subsystem analysis. Also add an explanatory note that this is being done in the initial build to avoid wasting money and that those sensors might be ordered at a later date.

For this subsystem it appears that every constraint applies to every sensor, so they all must be evaluated. The document will be easier to read if the numbering scheme is consistent. (If a particular constraint is not relevant to a particular sensor, just say that so the reader knows it's not a mistake.)

The justification for removing the Lead Monoxide sensor is invalid. Which pollutants are measured is a system-level decision which should have been made a long time ago. However, I will agree with removing it.

The two parts, ULPSM... and A02P4... are still not clear. Also does the word "below" mean it is physically mounted below, meaning in the direction that the gravity force is acting, the other part? If not, maybe a different wording will make this clear.

Yes, remove that reference because it is trivial as noted in the prior comment.

Reply:
Will do.

Okay, I will try and restructure the analysis into a numbered scheme. For points that don't follow a constraint I'll leave bulleted.

Well, based on the Project Proposal, there were pollutants stated that were optional but not mandatory. Sure I agree that it was a late change, but I was not aware that I had to look for every sensor stated until this week.

I understand the confusion. "The ULPSM-IAQ 968-008 will be connected using header pins under the A02P43B7001, which will communicate with the Microcontroller." I will add this for clarification.

Edited.

Reply:
I think most of these have been addressed but will need to look at them again after other parts have been updated.

Is there any sensor, other than the expensive one, that outputs analog current? If not, how will the analog current part of the interface subsystem be tested?

When evaluating sensors be sure to look at voltage tolerances and show that they are compatible with the tolerance requirements given in the constraints.

Reply:
That is true, that is the only analog current sensor I have found. To be fair, it was quite difficult to find a sensor output of that type that is included within the list of pollutants given in the 1st constraint. That is another reason why the sensors were not included in the budget from its various price ranges.

That should be shown in the analysis portion of the document.

Reply:
Be sure to evaluate every constraint. If it is not relevant state that and give a reason why. Also check the numbering as they don't appear to match up in all cases (were some constraint numbers changed?).

Reply:
Analysis points not supported by a constraint are for source code only. Therefore, I added "(see figure X)" for each source code material. The constraint numbers have been changed, I will update the analysis section with the corresponding constraints list.

The MIKROE-1628 is an analog voltage sensor. Not sure how it is conflicting with the classifications?

As mentioned yesterday, it was difficult for me to find an analog current sensor that detects pollutants in general. The detector and PCB module listed in the BOM are the closest things I could find to meet the specifications. I believe you mentioned yesterday that these types of sensors may have to be annulled from the list because of the pricing? I could leave a note by the BOM mentioning that.